cirq-google/cirq_google/experimental/noise_models/calibration_to_noise_properties.py

```diff
@@ -0,0 +1,106 @@
+import cirq_google
+import numpy as np
+from cirq.devices.noise_properties import NoiseProperties
+
+
+def _xeb_fidelity_to_decay_constant(xeb_fidelity, num_qubits=2):
+    # Converts from XEB Fidleity to depolarization decay constant
+    if xeb_fidelity is not None:
+        N = 2 ** num_qubits
+        return 1 - (1 - xeb_fidelity) / (1 - 1 / N)
+    return None
+
+
+def _rb_average_error_to_decay_constant(rb_average_error, num_qubits: int = 1):
+    # Converts from randomized benchmarking average error to depolarization decay constant
+    if rb_average_error is not None:
+        N = 2 ** num_qubits  # Dimension of Hilbert space
+        return 1 - rb_average_error / (1 - 1 / N)
+    else:
+        return None
+
+
+def _rb_pauli_error_to_decay_constant(rb_pauli_error, num_qubits: int = 1):
+    # Converts from randomized benchmarking pauli error to depolarization decay constant
+    if rb_pauli_error is not None:
+        N = 2 ** num_qubits  # Dimension of Hilbert space
+        return 1 - rb_pauli_error / (1 - 1 / N ** 2)
+    else:
+        return None
+
+
+def _within_tolerance(val_1, val_2, tolerance):
+    # Helper function to check if two values are within tolerance
+    if val_1 is None or val_2 is None:
+        return True
+    return abs(val_1 - val_2) <= tolerance
+
+
+def _unpack_from_calibration(metric_name, calibration):
+    # Gets the average (over all qubits) of each metric
+    # TODO: Add support for per-qubit noise
+    if metric_name in calibration.keys():
+        return np.mean([value for qubit, value in calibration[metric_name].items()])
+    else:
+        return None
+
+
+def noise_properties_from_calibration(
+    calibration: cirq_google.Calibration, validate: bool = True, tolerance: float = 0.01
+):
+    """Translates between a Calibration object and a NoiseProperties object1
+    The NoiseProperties object can then be used to create a NoiseModel for a simulator.
```

Comment:
This should link or reference the `NoiseModelFromProperties` class as an indication of how to do this conversion.